ui/lib/core/addon/components/toolbar-link.js

```diff
@@ -16,13 +16,21 @@ import Component from '@glimmer/component';
  * ```
  *
  * @param {array} params - Array to pass to LinkTo
- * @param {string} type - Use "add" to change icon
+ * @param {string} type - Use "add" to change icon to plus sign, or pass in your own kind of icon.
  * @param {boolean} disabled - pass true to disable link
  * @param {string} disabledTooltip - tooltip to display on hover when disabled
  */
 
 export default class ToolbarLinkComponent extends Component {
   get glyph() {
-    return this.args.type == 'add' ? 'plus' : 'chevron-right';
+    // not ideal logic, but updating old system to allow for other kinds of icons
```

Comment:
I agree on separate PR as a nice cleanup to address why we are just changing the word 'add' to 'plus' 😅 

But in the interim the following feels easier to read to me
```
get glyph() {
    const { type } = this.args;
    if (!type) return 'chevron-right';
    return type === 'add' ? 'plus' : type;
}

Reply:
Thank you for the cleanup. You rock at the tight logic statements.